examples/carrots.py

```diff
@@ -51,16 +51,12 @@ def private_mean(self, privacy_budget: float) -> dp.StatusOrO:
         return x.result(list(self._df["carrots_eaten"]))
 
     def private_count_above(self, privacy_budget: float, limit: int) -> dp.StatusOrO:
-        pass
-
-    def private_mean(self, privacy_budget: float) -> dp.StatusOrO:
-        pass
-
-    def private_count_above(self, privacy_budget: float, limit: int) -> dp.StatusOrO:
-        pass
+        x = dp.CountInt(privacy_budget)
+        return x.result(list(self._df[self._df.carrots_eaten > limit]["carrots_eaten"]))
 
     def private_max(self, privacy_budget: float) -> dp.StatusOrO:
-        pass
+        x = dp.Max(privacy_budget, 0, 150)
```

Comment:
Please add a comment on what 0 and 150 represent

Reply:
They are the search bounds, default values are numeric integers limits which gives a very inconsistent value.
Should I add a comment saying these are the search bounds ?

Reply:
Yes, please do that. 
Also, can you please add it in other part of code if it's not present?

Reply:
Okay, will do that

Reply:
I've added the comment, the private sum requires similar bounds but we haven't ported that yet.